job_definitions/docker_base_images.yml

```diff
@@ -1,6 +1,8 @@
 - job:
-    name: build-and-push-base-images
-    display-name: Build and push Digital Marketplace docker base images
+    name: docker-base-images
+    display-name: Docker base images
```

Comment:
Aw I like having a verb in our Jenkins jobs. How about `Refresh Docker base images`? Hmm maybe not, that sounds like they're having a spa day...

Reply:
renew?
regenerate? (Too TARDIS?)

Reply:
Rejuvenate!